nx-X11/programs/Xserver/hw/nxagent/Atoms.c

```diff
@@ -53,6 +53,11 @@
 #undef  TEST
 #undef  DEBUG
 
+#ifdef DEBUG
+/* for validateString() */
+# include "Utils.h"
```

Comment:
please remove the whitespace; this looks ugly and I don't think we use it this way anywhere else.